numpy/lib/npyio.py

```diff
@@ -412,11 +407,7 @@ def load(file, mmap_mode=None, allow_pickle=False, fix_imports=True,
         # result can similarly silently corrupt numerical data.
         raise ValueError("encoding must be 'ASCII', 'latin1', or 'bytes'")
 
-    if sys.version_info[0] >= 3:
-        pickle_kwargs = dict(encoding=encoding, fix_imports=fix_imports)
-    else:
-        # Nothing to do on Python 2
-        pickle_kwargs = {}
+    pickle_kwargs = dict(encoding=encoding, fix_imports=fix_imports)
 
     # TODO: Use contextlib.ExitStack once we drop Python 2
```

Comment:
Is a cleanup here on your list of things still TODO?

Reply:
The other one get's used three times and I thought shouldn't be cleaned up, but this one should be. Fixed now added a note to cleanup someother parts of this file.